app/services/storage.js

```diff
@@ -94,14 +111,20 @@ export default Ember.Service.extend(Ember.Evented, {
   },
 
   setupRemoteStorage() {
+    const remoteStorage = this.get('remoteStorage');
```

Comment:
No. But actually the `setup()` call to configure it should be there as well. Then we can get rid of the whole `app/instance-initializers/storage.js`.